events/importer/base.py

```diff
@@ -317,6 +317,27 @@ def info_make_link_id(info):
                     link_obj.save()
                 obj._changed = True
 
+        extension_data = info.get('extension_course')
+        if extension_data is not None:
+            from extension_course.models import Course
+
+            try:
+                course = obj.extension_course
+                course._changed = False
+                self._set_field(course, 'enrolment_start_time', extension_data['enrolment_start_time'])
+                self._set_field(course, 'enrolment_end_time', extension_data['enrolment_end_time'])
+                course_changed = course._changed
```

Comment:
why do you need to juggle these `course._changed` and `course_changed` like that? 🤔

Reply:
I mean:
```python
course._changed = False
...
course_changed = course._changed
```

could just be:
```python
course_changed = False
```

Reply:
ahhhhh, nevermind. I understand now.